frontend/src/pages/Home/index.tsx

```diff
@@ -29,22 +32,6 @@ export default function HomePage() {
           </Link>
         </Row>
       </Column>
-
-      <Column as="article" className={styles.secondArticle}>
```

Comment:
If we delete this part (I agree with that) we should also remove associated unused styles in `./styles.css.ts`